docs/source/advanced/authentication.mdx

```diff
@@ -74,4 +74,50 @@ val apolloClient = ApolloClient.Builder()
     .build()
 ```
 
+### Updating your WebSocket token
+
+When using authenticated subscriptions over WebSockets, you might want to refresh your token. Because it was invalidated or simply because it expired.
```

Comment:
(nit)
```suggestion
When using authenticated subscriptions over WebSockets, you might want to refresh your authentication token because it was invalidated or simply because it expired.
```